tools/state-parts-dump-check/src/cli.rs

```diff
@@ -19,8 +19,9 @@ use std::path::PathBuf;
 use std::sync::Arc;
 use std::thread::sleep;
 use std::time::{Duration, Instant};
+use tokio::time::timeout;
 
-const MAX_RETRIES: u32 = 3;
+const MAX_RETRIES: u32 = 5;
```

Comment:
Please add a comment why we shouldn't set this number to a very large number, like `1_000_000`.